notifications_api_common/viewsets.py

```diff
@@ -236,3 +236,13 @@ class NotificationViewSetMixin(
     NotificationCreateMixin, NotificationUpdateMixin, NotificationDestroyMixin
 ):
     pass
+
+
+add_autoretry_behaviour(
+    NotificationMixin.send_notification,
+    autoretry_for=(
+        NotificationException,
+        requests.RequestException,
+    ),
+    retry_jitter=False,
+)
```

Comment:
this looks like... not-so-elegant monkeypatching. Can we not apply this as a decorator on the actual method instead?